src/Arcus.Templates.ServiceBus.Topic/Program.cs

```diff
@@ -82,22 +84,28 @@ public static IHostBuilder CreateHostBuilder(string[] args)
         }
 #if Serilog
         
-        private static void UpdateLoggerConfiguration(
-            HostBuilderContext hostContext,
-            LoggerConfiguration config)
+        private static async Task ConfigureSerilogAsync(IHost host)
         {
-            config.MinimumLevel.Debug()
-                  .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
-                  .Enrich.FromLogContext()
-                  .Enrich.WithVersion()
-                  .Enrich.WithComponentName("Service Bus Topic Worker")
-                  .WriteTo.Console();
+            var secretProvider = host.Services.GetRequiredService<ISecretProvider>();
+            string connectionString = await secretProvider.GetRawSecretAsync(ApplicationInsightsConnectionStringKeyName);
             
-            var instrumentationKey = hostContext.Configuration.GetValue<string>(ApplicationInsightsInstrumentationKeyName);
-            if (!string.IsNullOrWhiteSpace(instrumentationKey))
+            var reloadLogger = (ReloadableLogger) Log.Logger;
+            reloadLogger.Reload(config =>
             {
-                config.WriteTo.AzureApplicationInsights(instrumentationKey);
-            }
+                config.MinimumLevel.Debug()
```

Comment:
Same here regarding the loglevel

Reply:
Will be done in https://github.com/arcus-azure/arcus.templates/issues/624